### ISSUE TITLE
Starts with blank window

### ISSUE THREAD

**Issue by Peter-Barrow** [JerryI/wolfram-js-frontend#72]
I have encountered the same issue as in #58. This behaviour exists when running from a clone of the repository, .deb and the appimage. This is on Ubuntu 22.04 with Wolfram Engine 13.0.1.

**Comment by JerryI** [comment]
Hello @Peter-Barrow ,
Could you please share `2minutesLog.txt` located in AppData or in the root folder of the repo?

PS: You can also do it privately by sending it to my email krikus.ms@gmail.com

**Comment by JerryI** [comment]
Thank you for your log file!
There are two options on how to fix it:

1. make `git pull` from this repository, I have replaced a buggy `PingTime` function with a direct HTTP request. If there is an internet connection, it will fetch all libraries and after than can work fully offline. Please, make sure before starting, that folders `Packages`, `wl_packages` and a file `wl_packages.wl` are removed. So that it guarantees a "clean" installation.
2. I [have uploaded a copy](https://github.com/JerryI/wolfram-js-frontend/releases/download/1.0.7/wolfram-js-frontend-with-all-libraries-included.zip) of the repository with all libraries included (the most recent versions). In principle, it should work just from the console in the same way, but it will detect `wl_packages.wl` file and won't download anything. 

Hope it will help. I definitely need to improve this package manager...

**Comment by Peter-Barrow** [comment]
Thanks for the help here. I now have it running from the repo, but with one caveat. When I run `wolframscript -f Scripts/run.wls` it goes through what looks like the expected build process but it never and `./Packages` gets created, however, at no point is `wl_pacakges` created. When run like this the previous error persists where the ui errors trying to connect to the backend.

### A potential solution...
I just copied `wl_packages` from the zip you uploaded into my local copy of the repo and everything appears to run as expected.

<details>
<summary><b>Running</b></summary>

![image](https://github.com/JerryI/wolfram-js-frontend/assets/24245170/6182b4ce-c481-4dc3-b3cb-3333239b67d7)

</details>

<details>
<summary><b>Evaluates</b></summary>

![image](https://github.com/JerryI/wolfram-js-frontend/assets/24245170/587c92cf-6356-4842-abdd-82fabacadaee)

</details>


It is also worth pointing out that the zipped version would not run properly on my machine running like so:
![image](https://github.com/JerryI/wolfram-js-frontend/assets/24245170/c1eb0dd8-8590-4dfb-8855-0494db593342)

**Comment by Peter-Barrow** [comment]
I have a new solution that hasn't required me copying any files over from the .zip archive. Now, it might not be best practices but replacing `URLResponseTime' with `URLFetch` in `Services/LPM.wl` has it initialise without issue

**Comment by JerryI** [comment]
I tested it again on a different machine, it did work well. If you experience any other problem, please, open another issue here.
